microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,180 @@
+
+Previously in Microsoft Edge, selecting **Edit and resend** for a network request from the Network tool may not have reliably opened the Network Console tool and prepopulated it with the values from the request. 
+
+In Microsoft Edge 105, this issue has been fixed. The **Edit and resend** now prepopulates the Network Console tool with the values for the network request that you are looking to resend. You can continue to modify these values before sending the request from the Network Console tool. Thank you for sending us your feedback about this issue!
+
+Right-click the network request that you want to change and resend, and then select *Edit and Resend*.
+
+[1](./devtools-105-images/edit-and-resend.msft.png
+
+In the Network Console, edit the network request information, and then click the *Send* button.
+[2](./devtools-105-images/networkconsole-edit.msft.png
+
+
+See also:
+* [Compose and send web API requests using the Network Console tool](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/network-console/network-console-tool#starting-from-the-network-tool)
+
+<!-- ====================================================================== -->
+## Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View
+
+<!-- Rank 3 -->
+<!-- Title: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View -->
+<!-- Subtitle: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View. -->
+
+Several improvements to [Focus Mode](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode) in existing location controls for DevTools have been made, including a new control for changing the orientation of Quick View.
+
+The Settings menu now provides options for changing DevTools between docked locations and a separate window. This menu also contains settings for how the Activity Bar is displayed. Starting in Edge 105, these options can be navigated and changed more quickly. The previous submenus have been replaced with toggle buttons. Additionally, improved contrast means the option you have selected is more easily visible at a glance.
+
+Before:
+
+![1 before](./devtools-105-images/before-docking-menu.msft.png)
+
+After:
+
+![1 after](./devtools-105-images/after-docking-menu.msft.png)
+
+<!-- 
+Screenshot 1, "before" in Edge 104: Ensure Focus Mode is enabled. Open Settings menu, then navigate to the docking submenu to show options.
+Screenshot 1, "after" in Edge 105: Ensure Focus Mode is enabled. Open Settings menu to show DevTools docking location and Activity Bar location settings. -->
+
+You can now change the orientation of the Quick View panel as well. To change Quick View to a vertical split, select the location toggle (**Dock Quick View to the right**). Selecting this toggle a second time will return Quick View to a horizontal split.
+
+![2](./devtools-105-images/quickview-console.png) 
```

Comment:
Create actual final-quality screenshot, crisper, red boxes instead of rough arrows

Reply:
I fixed the screenshots